### PR TITLE
Rerender whole website using CRAN versions

### LINKED ISSUE #261: Search all of tidymodels needs to be used to index version 1.0.0 of packages
I found this problem when trying to find `control_last_fit()` which was added in tune 1.0.0

Screen Shot 2022-09-14 at 3 45 02 PM

### PR DESCRIPTION
**Pull request by EmilHvitfeldt**
This PR will close https://github.com/tidymodels/tidymodels.org/issues/261

Deploy preview: https://deploy-preview-263--tidymodels-org.netlify.app/

### CODE REVIEW

File: content/start/resampling/index.markdown [hunk 1]
```diff
@@ -40,16 +40,19 @@ Let's use data from [Hill, LaPan, Li, and Haney (2007)](http://www.biomedcentral
...
+#> 2 Train PS        134.       819    31.9  207.     69.9   164.      1.26   0.797
+#> 3 Train WS        107.       431    28.0  116.     63.9   107.      1.05   0.935
+#> 4 Train PS         69.2      298    19.5  102.     28.2    31.0     1.20   0.866
+#> 5 Test  PS          2.89     285    24.3  112.     20.5    40.6     1.11   0.957
+#> # … with 2,014 more rows, 48 more variables: diff_inten_density_ch_1 <dbl>,
+#> #   diff_inten_density_ch_3 <dbl>, diff_inten_density_ch_4 <dbl>,
+#> #   entropy_inten_ch_1 <dbl>, entropy_inten_ch_3 <dbl>, …, and abbreviated
+#> #   variable names ¹​area_ch_1, ²​avg_inten_ch_1, ³​avg_inten_ch_2,
+#> #   ⁴​avg_inten_ch_3, ⁵​avg_inten_ch_4, ⁶​convex_hull_area_ratio_ch_1,
+#> #   ⁷​convex_hull_perim_ratio_ch_1
```
Comment (hfrick) [inline content/start/resampling/index.markdown]: this would be a lot nicer with full column names imo

**Review by hfrick — COMMENTED** [review]
Thanks for updating that! Some of it was quite old. I noticed two main things:

In several places the numbers change, e.g. predictions, performance metrics, etc. From reading over it, I think the changes are not large and don't change the story. But I wanted to ask if those changes are expected. If not, do we mind?

The other thing I noticed is the abbreviated column names of tibbles. Especially for the `cells` data, I think it would be nice to set the corresponding pillar option to allow it to print the full names. That option is also mentioned in this pillar issue https://github.com/r-lib/pillar/issues/582

File: content/start/resampling/index.markdown (hunk 1 above)
Comment (EmilHvitfeldt) [reply content/start/resampling/index.markdown]: Agree, I'll start another run and hopefully have it ready tomorrow 🤞

**Comment by EmilHvitfeldt** [comment]
With regards to the changing numbers, as far as I can tell none of them are major/changing. Also there have been some small changes like; different {dials} ranges changing, {rsample} bug fix etc etc. I think this is good.

Moving forward I'm planning to do more regular re-rendering so we catch these changes more individually.

File: content/start/resampling/index.markdown (hunk 1 above)
Comment (hfrick) [reply content/start/resampling/index.markdown]: How long does it take to render everything? 👀

Comment (EmilHvitfeldt) [reply content/start/resampling/index.markdown]: 5-6 hours, sometimes less if I'm lucky 😅

**Comment by EmilHvitfeldt** [comment]
I rerendered everything a couple of more times, each time finding post that wasn't seeded. Everything should be seeded now and runs the same every time

File: content/start/resampling/index.markdown (hunk 1 above)
Comment (hfrick) [reply content/start/resampling/index.markdown]: 🙀

**Comment by hfrick** [comment]
Thanks so much for persevering through all the re-renders!

**Pull request merged**